packages/notification-spa/src/app/app.gql.ts

```diff
@@ -1,41 +1,79 @@
 import gql from 'graphql-tag';
 
-export const listNotificationItems = gql`
-query listNotificationItems {
-    notificationItems {
-        name,
-        title
+const notificationConfigFragment = gql`
+fragment notification on NotificationConfig {
+  id
+  configID
+  template
+  defaultLink
+  channel
+  type
+  action
+  targets
+  source {
+    _id
+    name
+    icon
+    active
+    owners {
+      _id
+      name
     }
+  }
+  createdBy
+  createdOn
+  updatedBy
+  updatedOn
 }
 `;
 
-export const manualNotification = gql`
-mutation manualNotificaion (
-    $input: manualNotification
-    ) {
-        manualNotification (
-            input: $input
-        ) {
-            name,
-            channel,
-            trigger,
-            infomation,
-        }
-    }
+export const getNotificationConfigBy = gql`
+${notificationConfigFragment}
+query GetNotificationConfigsBy ($input: NotificationConfigInput!) {
+  getNotificationConfigsBy(notificationConfig: $input) {
+    ...notification
+  }
+}
 `;
 
-export const notificationFormData = gql`
-mutation notificationFormData (
-    $input: notificationFormData
-    ) {
-        notificationFormData (
-            input: $input
-        ) {
-            _id,
-            name,
-            channel,
-            trigger,
-            infomation,
-        }
-    }
-`;
\ No newline at end of file
+export const deleteNotificationConfig = gql`
+${notificationConfigFragment}
+mutation DeleteNotificationConfig ($id: String!) {
+  deleteNotificationConfig(id: $id) {
+    ...notification
```

Comment:
You need to use a fragment of type `NotificationConfigRaw` for mutations.
As that provides a string type for the `source` field. (Check #460)

Reply:
Added NotificationConfigRaw for fragment